echoregions/tests/test_regions2d.py

```diff
@@ -617,12 +686,18 @@ def test_within_transect(
         da_Sv=da_Sv_fixture, transect_dict=transect_dict
     ).compute()
 
+    # Check M dimensions
+    assert M.sizes == {"depth": 3955, "ping_time": 1681}
+
     # This entire .zarr file should be covered by the single start and end transect period
     # found in the EVR file, so the only values listed should be 1, implying everything is
     # within-transect.
     assert len(list(np.unique(M.data))) == 1
     assert list(np.unique(M.data))[0] == 1
 
+    # Test number of 1 values
+    assert np.unique(M.data, return_counts=True)[1][0] == 6648355
```

Comment:
I was thinking to test the actual shape, in case in future we transpose something.